systers_portal/meetup/forms.py

```diff
@@ -0,0 +1,35 @@
+from django import forms
+
+from common.forms import ModelFormWithHelper
+from common.helpers import SubmitCancelFormHelper
+from meetup.models import Meetup
+from users.models import SystersUser
+
+
+class AddMeetupForm(ModelFormWithHelper):
+    """Form to create new Meetup. The created_by and the meetup_location of which meetup belong to
+    are expected to be provided when initializing the form:
+
+    * created_by - currently logged in user
+    * meetup_location - to which Meetup belongs
+    """
+    class Meta:
+        model = Meetup
+        fields = ('title', 'slug', 'date', 'time', 'venue', 'description')
+        widgets = {'date': forms.DateInput(attrs={'type': 'text', 'class': 'datepicker'}),
+                   'time': forms.TimeInput(attrs={'type': 'text', 'class': 'timepicker'})}
+        helper_class = SubmitCancelFormHelper
```

Comment:
Since we don't have meetup location page ready, for now I have added about meetup location page with `helper_cancel_href`.